kubo-config.example.sh

```diff
@@ -9,4 +9,9 @@ ipfs config --json Gateway.PublicGateways '{
 		"InlineDNSLink": true,					
 		"Paths": ["/ipfs", "/ipns", "/api"]		
 	}											
-}'
\ No newline at end of file
+}'
+
+IPFS_NS_MAP=$(cat ./dnslinks.json | jq -r 'to_entries | map("\(.key).example.com:\(.value)") | join(",")')
```

Comment:
If we export, then I think we could call this script as `source kubo-config.example.sh` to get the env var in parent shell.

```suggestion
export IPFS_NS_MAP=$(cat ./dnslinks.json | jq -r 'to_entries | map("\(.key).example.com:\(.value)") | join(",")')
```

Reply:
👍 good catch, that should make the CI test simpler, I'll update the script.